src/api/spec/factories/project.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :project do
-    name Faker::Internet.domain_word
+    sequence(:name){|n| "#{Faker::Internet.domain_word}#{n}" }
```

Comment:
To make login unique! From faker gem readme

`
NOTE: While Faker generates data at random, returned values are not guaranteed to be unique.
`